.github/workflows/test-integrations-common.yml

```diff
@@ -36,6 +36,8 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
+        env:
+          PIP_TRUSTED_HOST: "pypi.python.org pypi.org files.pythonhosted.org"
```

Comment:
What is the purpose of adding this line?

Reply:
See https://github.com/getsentry/sentry-python/pull/3341/files#r1692923153